ballerina/types.bal

```diff
@@ -115,7 +126,20 @@ public type SearchRequest record {
     float[][]|float[] vectors;
     int topK;
     string filter?;
-    string outputFields?;
+    string[] outputFields?;
+};
+
+# Represents the request for the query operation.
+#
+# + collectionName - The name of the collection to search data from
+# + partitionName - The name of the partition to search data from
+# + filter - The filter to search for
+# + outputFields - The fields to return
```

Comment:
shall we move the doc comments to field level?

Reply:
check other places

Reply:
Fixed in [d0eb77f](https://github.com/ballerina-platform/module-ballerinax-milvus/pull/7/commits/d0eb77f20e5831292ef27fe8ed9a044035f9a07c)